opensrp-giz-malawi/src/main/res/layout-w900dp/navigation_item.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
For which devices does this apply? Do our deliverables cover multiple screen sizes and/or devices? If for tablet only, does this apply well on what we expect to be used on the gro

Below is from https://developer.android.com/training/multiscreen/screensizes.
<img width="855" alt="Screen Shot 2019-07-22 at 4 13 58 PM" src="https://user-images.githubusercontent.com/31766075/61635225-c36b2c80-ac9b-11e9-8b13-371ce3c40ea3.png">

Reply:
fixed